.github/workflows/lint-pr-title.yml

```diff
@@ -0,0 +1,56 @@
+name: "Lint PR title"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - labeled
+      - unlabeled
```

Comment:
Curious why you are triggering for labels

Reply:
We can enable/disable this check with the `ignore-semantic-pull-request` label.

Reply:
thats a pretty neat feature actually!!